### ISSUE TITLE
Marked a Movie as Watched & Rated on Trakt App but Won't Sync to Plex?

### ISSUE THREAD

**Issue by jeff15110168** [trakt/Plex-Trakt-Scrobbler#623]
So I watched a movie in theaters back in April (Everything Everywhere All At Once), I marked it as watched and rated it in the Trakt app right after leaving the theater. Same movie recently got added to my Plex but the watched / rating won't sync even though I have it set to Synchronize (Push + Pull). I've run a full sync in Kitana and it says successful but it still won't sync. This seems specific to movies that I rate in Trakt before they are added to Plex, as I otherwise see ratings synced from Trakt to Plex for movies that existed in my library before I add the rating in Trakt app. Any idea what could be causing this / potential fix?

Also tried to access via http://trakt-for-plex.github.io/configuration/#/connect but it says "Unable to find valid connection" for some reason now.

![image](https://user-images.githubusercontent.com/9892738/168964849-7615435e-cd7f-4dde-bd7e-8b596d4580b2.png)

**Comment by the-jeffski** [comment]
I fix this periodically by doing a 'Pull from Trakt.tv'  in Kitana rather than a sync.  It then should overwrite with whats in Trakt.  I found if you have run after library updates enabled you don't get this issue.

**Comment by jeff15110168** [comment]
> I fix this periodically by doing a 'Pull from Trakt.tv' in Kitana rather than a sync. It then should overwrite with whats in Trakt. I found if you have run after library updates enabled you don't get this issue.

Interesting. What do you mean by "running after library updates"?

**Comment by the-jeffski** [comment]
The Sync after library updates option a bit further down.

**Comment by jeff15110168** [comment]
i see - i think i've always had that box checked in kitana settings. So running "Pull from trakt.tv" when there's a mismatch between plex not pulling ratings is the solution for now?

![image](https://user-images.githubusercontent.com/9892738/172879566-f457024d-7a58-4cfc-96d8-e22d77ddfe3a.png)

**Comment by the-jeffski** [comment]
Yes, thats what I do. Basically it then pulls everything and and watched
get updated regardless of when
